### DISCUSSION TITLE
https://yuyichen.space/#/post/5

### DISCUSSION THREAD

**Discussion by giscus** [yuyichen/blog#11] [Announcements]
# https://yuyichen.space/#/post/5

羽衣尘的个人技术博客

https://yuyichen.space/#/post/5

**Comment by hao-lee** [comment]
我有个大胆的想法，通过不断的请求唠唠动态，抓取发帖人和回帖人，岂不是可以做个爬虫，按照10秒爬一贴的速度，一个月能爬40万用户。根据青藤官网的数据，两个月相当于拖库...

**Reply by yuyichen** [reply]
想法是不错，但是爬虫这东西，你也知道，大概率不合规合法，搞不好会被请喝茶，而且爬到这些数据有价值的部分也不多，用户信息都是脱敏的

**Reply by hao-lee** [reply]
🥲

**Reply by xuebuaa** [reply]
这可不兴爬啊宝友，日子过得是越来越有判头了Ծ‸Ծ

**Reply by hao-lee** [reply]
> 这可不兴爬啊宝友，日子过得是越来越有判头了Ծ‸Ծ

@xuebuaa 我都没意识到这点，感谢老哥们提醒...不然就真刑了

**Reply by hao-lee** [reply]
我看了点资料，爬虫只要不把对方服务器搞崩，不用来盈利，不泄漏爬到的数据，应该不会被请喝茶的。到时候搞个国外vps把爬虫布上去。我之所以想搞爬虫主要还是这软件太坑了，手握大量资源却每天只给你推荐10个，而且还没办法做细粒度筛选，找个合适的人真是太难了，真要等到猴年马月去。

**Reply by yuyichen** [reply]
还是劝你别折腾了，爬到了数据，然后呢，又不能配对，而且我感觉很大比例的人根本不发动态

**Reply by lh4111** [reply]
给你点关注了兄弟

**Comment by yuyichen** [comment]
体验站点应该是被官方发现并使用了反制措施，这个情况我一开始就预料到了，网站相关功能我已做了下线处理，如果想继续体验，可以参照本文思路自行实现

**Reply by xiewh7** [reply]
寄

**Reply by yuyichen** [reply]
？

**Comment by YWD** [comment]
老哥，是不是抓不了包了，用charles代理，打开小程序就提示开代理了，让退出

**Reply by yuyichen** [reply]
抱歉，才看到消息，我失业一年，青藤也注销了。看到你这个问题，我的想法是要么你尝试下其他拦截请求的方法，要么试下反编译小程序源码看看他们怎么做到的。最后祝你早日找到对象哈，平台只能是辅助，还是得努力提升自身才是正途。

**Comment by hao-lee** [comment]
> 抱歉，才看到消息，我失业一年，青藤也注销了。看到你这个问题，我的想法是要么你尝试下其他拦截请求的方法，要么试下反编译小程序源码看看他们怎么做到的。最后祝你早日找到对象哈，平台只能是辅助，还是得努力提升自身才是正途。

老哥怎么失业了，现在怎么样了

**Reply by yuyichen** [reply]
去年年底整个部门被裁了，我一个大龄前端，长时间找不到工作，只能回老家躺平了。
